crates/sdk/tests/define.rs

```diff
@@ -2327,10 +2330,10 @@ async fn permissions_checks_define_user_root() {
 	]);
 
 	// Define the expected results for the check statement when the test statement succeeded and when it failed
-	let check_results = [
-        vec!["{ accesses: {  }, namespaces: {  }, nodes: {  }, users: { user: \"DEFINE USER user ON ROOT PASSHASH 'secret' ROLES VIEWER DURATION FOR TOKEN 15m, FOR SESSION 6h\" } }"],
-		vec!["{ accesses: {  }, namespaces: {  }, nodes: {  }, users: {  } }"]
-    ];
+	let parallelism = available_parallelism().unwrap();
+	let access1 = format!("{{ accesses: {{  }}, namespaces: {{  }}, nodes: {{  }}, system: {{ parallelism: {parallelism}}}, users: {{ user: \"DEFINE USER user ON ROOT PASSHASH 'secret' ROLES VIEWER DURATION FOR TOKEN 15m, FOR SESSION 6h\" }} }}");
```

Comment:
Just a nitpick, but this probably should be called `user1` and `user2` instead. The more generic `check1` and `check2` names like you used in the tests for `REMOVE` also seem fine to me, if you don't want to think about the naming 👍